securedrop/Makefile

```diff
@@ -0,0 +1,37 @@
+TAG ?= $(shell git rev-parse HEAD)
+IMAGE = securedrop
+
+.PHONY: assets
+assets:
+	sass
+
+.PHONY: images
+images:
+	docker build $(EXTRA_BUILD_ARGS) -t $(IMAGE):$(TAG) .
+	# combine dockerfiles but strip out first COPY for better cache utilization
+	cat Dockerfile* |sed '/#skip-if-test$$/N;/#skip-if-test/d' | docker build $(EXTRA_BUILD_ARGS) -f - -t $(IMAGE)-test:$(TAG) .
```

Comment:
Is this really necessary? I'd prefer breaking these dockerfiles over two separate files without this concatenation action. In other words if it takes a little longer for builds i'd rather have that versus this logic.

Reply:
Agreed with @msheiny—although there's sophisticated cache optimization with the concatenation approach, it may pose problems for maintainability, particularly from new contributors. I'm fine trying out this approach and revising later, though, since the changes in this PR get us pretty far ahead of where we are now.

Reply:
I concur that this is probably a little too fancy. I can simplify this and settle on one. I was thinking ahead to what publishing official images might look like and how cache sharing could assist in that situation. I'll back this out and it can be revisited in the future.

Reply:
pushed this change.